newIDE/app/src/MainFrame/index.js

```diff
@@ -2033,6 +2051,14 @@ const MainFrame = (props: Props) => {
       : () => {}
   );
 
+  const onUserflowRunningUpdate = () => {
+    // Userflow dialog has a variable exported which knows if the
+    // onboarding is running or not.
+    // To ensure all components are aware of this variable when it changes,
+    // we need to force a re-render.
+    forceUpdate();
```

Comment:
Just a though: should we write `const onUserflowRunningUpdate = forceUpdate`? I was afraid it might be invisible to the eye, but I guess it's okay